src/System.Collections/tests/Performance/Perf.Dictionary.cs

```diff
@@ -193,5 +193,356 @@ public void ContainsKey(int size)
                         dict.ContainsKey(key); dict.ContainsKey(key); dict.ContainsKey(key);
                     }
         }
+
+        [Benchmark]
+        [InlineData(1)]
+        [InlineData(10)]
+        [InlineData(1000)]
+        public static void Remove_ValueType(long size)
+        {
+            s_collection1 = new Dictionary<long?, long?>();
+            long?[] items;
+
+            items = new long?[size * 10];
+
+            for (long i = 0; i < size * 10; ++i)
+            {
+                items[i] = i;
+                s_collection1.Add(items[i], items[i]);
+            }
+
+
+            foreach (var iteration in Benchmark.Iterations)
+                using (iteration.StartMeasurement())
+                    for (long i = 1; i < size; ++i)
+                        s_collection1.Remove(items[i]);
+        }
+
+        [Benchmark]
+        public static void Indexer_get_ValueType()
```

Comment:
Is this duplicative of the GetItem test above:
https://github.com/dotnet/corefx/pull/7152/files#diff-53f002bec73f1f8b96c66bcef4e13b18R83
or is it testing something different?